### PR TITLE
Set Piwik Site ID to theme

### PR DESCRIPTION
**Pull request by petzlux**
Record a visit using the right Piwik Site ID. This will also record a separate visit if user changes the Theme.

### CODE REVIEW

**Comment by petzlux** [comment]
@rmichaelis review and merge please!

File: geoportailv3/static/js/themeservice.js [hunk 1]
```diff
@@ -7,16 +7,42 @@ goog.require('app.Themes');
...
+  /**
+   * @const
+   * @private
+   * @type {Object}
+   */
+  this.piwikSiteIdLookup_ = {
+    'main': 18,
+    'tourisme': 7,
+    'emwelt': 8,
+    'eau': 6
```
Comment (rmichaelis) [inline geoportailv3/static/js/themeservice.js]: Don't we need a different id for tourism V2 and V3 and for the other sites ? (As we do for the main site)

File: geoportailv3/static/js/themeservice.js [hunk 2]
```diff
@@ -57,6 +83,15 @@ app.Theme = function(ngeoLocation, appThemes, appScalesService, maxExtent) {
  */
 app.Theme.prototype.setCurrentTheme = function(themeId, map) {
   this.currentTheme_ = themeId;
+
+  var piwikSiteId = this.piwikSiteIdLookup_[this.currentTheme_];
+  if (!goog.isDefAndNotNull(piwikSiteId)) {
+    piwikSiteId = this.piwikSiteIdLookup_[app.Theme.DEFAULT_THEME_];
```
Comment (rmichaelis) [inline geoportailv3/static/js/themeservice.js]: If the theme id is not found, we use the main theme.
Maybe would it be better to have a different site id ?

File: geoportailv3/static/js/themeservice.js (hunk 1 above)
Comment (rmichaelis) [inline geoportailv3/static/js/themeservice.js]: Could it be useful to already have all the existing themes defined here ?

**Comment by rmichaelis** [comment]
It works as expected.
Only a few non blocking comments.
You can merge if they are not relevant.

File: geoportailv3/static/js/themeservice.js (hunk 1 above)
Comment (jaykayone) [inline geoportailv3/static/js/themeservice.js]: for the different ids: we wont keep tourismev2 so no need

for the complete list i agree but we can also add it later

File: geoportailv3/static/js/themeservice.js (hunk 2 above)
Comment (jaykayone) [inline geoportailv3/static/js/themeservice.js]: has been like that for v2 so ok for me for now

**Pull request merged**